_integration-test/run.sh

```diff
@@ -130,6 +132,42 @@ cat debug.log
 echo '------------------------------------------'
 echo "${_endgroup}"
 
+echo "${_group}Test symbolicator works ..."
+SENTRY_ORG="${SENTRY_ORG:-sentry}"
+SENTRY_PROJECT="${SENTRY_PROJECT:-native}"
+SENTRY_TEAM="${SENTRY_TEAM:-sentry}"
+# First set up a new project if it doesn't exist already
+PROJECT_JSON=$(jq -n -c --arg name "$SENTRY_PROJECT" --arg slug "$SENTRY_PROJECT" '$ARGS.named')
+NATIVE_PROJECT_ID=$(sentry_api_request "api/0/teams/$SENTRY_ORG/$SENTRY_TEAM/projects/" | jq -r '.[]|select(.slug == "'"$SENTRY_PROJECT"'")|.id')
+if [ -z "${NATIVE_PROJECT_ID}" ]; then
+  NATIVE_PROJECT_ID=$(sentry_api_request "api/0/teams/$SENTRY_ORG/$SENTRY_TEAM/projects/" -X POST --data "$PROJECT_JSON" | jq -r '. // null | .id')
+fi
+# Set up sentry-cli command
+SCOPES=$(jq -n -c --argjson scopes '["event:admin", "event:read", "member:read", "org:read", "team:read", "project:read", "project:write", "team:write"]' '$ARGS.named')
+SENTRY_AUTH_TOKEN=$(sentry_api_request "api/0/api-tokens/" -X POST --data "$SCOPES" | jq -r '.token')
+SENTRY_DSN=$(sentry_api_request "api/0/projects/sentry/native/keys/" | jq -r '.[0].dsn.secret')
+# Then upload the symbols to that project (note the container mounts pwd to /work)
+SENTRY_URL="$SENTRY_TEST_HOST" sentry-cli upload-dif --org "$SENTRY_ORG" --project "$SENTRY_PROJECT" --auth-token "$SENTRY_AUTH_TOKEN" windows.sym
+# Get public key for minidump upload
+PUBLIC_KEY=$(sentry_api_request "api/0/projects/sentry/native/keys/" | jq -r '.[0].public')
+# Upload the minidump to be processed, this returns the event ID of the crash dump
+EVENT_ID=$(sentry_api_request "api/$NATIVE_PROJECT_ID/minidump/?sentry_key=$PUBLIC_KEY" -X POST -F 'upload_file_minidump=@windows.dmp' | sed 's/\-//g')
+# We have to wait for the item to be processed
```

Comment:
Unfortunately, self-hosted runs symbolicator with caches, and those can mean things aren't processed right away. I'm not aware of a way to flush the caches, the tests in Sentry say to run without them(!)

Reply:
> I'm not aware of a way to flush the caches

Did you ask in #discuss-ingest?

> the tests in Sentry say to run without them

Is that what we're doing here?

Reply:
No, we are running with caches enabled, since that is more true to how self-hosted runs. I can ask in #discuss-ingest but since the Sentry tests don't flush the cache, and there is no documentation about how to do so, I presume it isn't possible.

Reply:
The cache should not be persistent. It seems that there is no way to manually trigger a flush :/